src/helpers/dropdownOptions.js

```diff
@@ -0,0 +1,56 @@
+export const options = {
```

Comment:
Beginning of the select options will be able to easily expand so a user can build more dynamic boards in the future